### ISSUE TITLE
Missing regular expression delimiters are not reported

### ISSUE THREAD

**Issue by jaroslavtyc** [kalessil/phpinspectionsea#1244]
Missing or non-matching regexp delimiters:
--
Example of invalid but easily overlooked regexp because of missing delimiters:
![missing_delimiters_are_not_reported](https://user-images.githubusercontent.com/2290225/54518654-1e46ed00-4965-11e9-8d5c-cb58fa8a39ab.png)

_Didn't I missed something? Becase not a PHPStorm itself nor PHP inspections report warning caused by such invalid regexp._

For example this ```preg_match('~\d~')``` is valid, but this ```preg_match('\d')``` causes
> PHP Warning:  preg_match(): Delimiter must not be alphanumeric or backslash

and nothing is matched at all of course.

As I personally forget delimiters quite often, a simple IDE warning would save some of my nerves every day.

Desired result
--
Check if regexp containts plain text and such plain text begins **and** ends with the same character would be enought from my point of view and marking detected problem as a warning(?) (same level as PHP does).

---
Thanks a lot for your hard and usefull work.

**Comment by kalessil** [comment]
Implemented!
